easybuild/easyconfigs/t/ToFu/ToFu-1.4.0-intel-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,51 @@
+easyblock = 'PythonBundle'
+
+name = 'ToFu'
+version = '1.4.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/ToFuProject/tofu'
+description = """Tomography for Fusion."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+dependencies = [
+    ('Python', '3.6.4'),
+    ('matplotlib', '2.1.2', versionsuffix),
+]
+
+exts_default_options = {'source_urls': [PYPI_LOWER_SOURCE]}
+
+exts_list = [
+    ('docutils', '0.14', {
+        'checksums': ['51e64ef2ebfb29cae1faa133b3710143496eca21c530f3f71424d77687764274'],
+    }),
+    ('Polygon3', '3.0.8', {
+        'modulename': 'Polygon',
+        'checksums': ['382aecaf1b550134aa71cc9c0f880a244e0ac5273e05bf898f5a6cde5bff9e2c'],
+    }),
+    ('zope.interface', '4.6.0', {
+        'checksums': ['1b3d0dcabc7c90b470e59e38a9acaa361be43b3a6ea644c0063951964717f0e5'],
+    }),
+    ('DateTime', '4.3', {
+        'modulename': 'DateTime',
+        'checksums': ['5cef605bab8259ff61281762cdf3290e459fbf0b4719951d5fab967d5f2ea0ea'],
+    }),
+    ('tofu', version, {
+        'patches': ['ToFu-1.4.0-no-git.patch', 'ToFu-1.4.0-intel.patch'],
+        'checksums': [
+            'f572581a9440de844f25e04194d624c37382e2dc171ee164febad7b8ff8e57c5',  # tofu-1.4.0.tar.gz
+            '6aae61d5eef6b7389fdae3eaac7bc9d6e045a1e75f897209516c4f2605f14855',  # ToFu-1.4.0-no-git.patch
+            '96013dc65891eb9645ff7efed1190846b41cd837ebec0827436a8673ac4589c1',  # ToFu-1.4.0-intel.patch
+        ],
+    }),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
```

Comment:
This sanity_check_paths is the default for PythonBundle and can be removed

Reply:
Thanks a lot for your hints. I've incorporated them all and squashed the commit history. Hope it builds :)